source/Audio.cpp

```diff
@@ -110,6 +117,17 @@ namespace {
 	shared_ptr<Music> previousTrack;
 	int musicFade = 0;
 	vector<int16_t> fadeBuffer;
+	double musicVolume = 1.;
+	double playlistVolumeModifier = 0.;
```

Comment:
```suggestion
	// The volume modifier of the current track.
	double playlistVolumeModifier = 0.;
```
Should it be called `trackVolumeModifier`, since it's actually defined per track?

Reply:
ye probably